homeassistant/components/velux/light.py

```diff
@@ -35,6 +35,11 @@ class VeluxLight(VeluxEntity, LightEntity):
     _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
     _attr_color_mode = ColorMode.BRIGHTNESS
 
+    def __init__(self, node: LighteningDevice) -> None:
+        """Initialize VeluxLight."""
+        super().__init__(node)
+        self.node: LighteningDevice = node
```

Comment:
```suggestion
    node: LighteningDevice
```
Can you maybe try if this also fixes it?